controllers/cloud.redhat.com/providers/resource_cache.go

```diff
@@ -187,12 +189,19 @@ func (o *ObjectCache) Create(resourceIdent ResourceIdent, nn types.NamespacedNam
 		return fmt.Errorf("cannot create: ident store [%s] already has item named [%s]", resourceIdent, nn)
 	}
 
-	gvk, err := getKindFromObj(o.scheme, resourceIdent.GetType())
+	var gvk, obGVK schema.GroupVersionKind
```

Comment:
Ahh... Declare it first and then use it in the `if thing, err` pattern. :facepalm: for me.

Reply:
Yup :)